src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_utils.py

```diff
@@ -85,6 +87,19 @@ def confirm_all_ip_allow():
             'Unable to prompt for confirmation as no tty available. Use --yes.') from e
 
 
+def confirm_enable_entra_auth():
+    try:
+        if not prompt_y_n(ENABLE_ENTRA_AUTH_MESSAGE):
```

Comment:
need to provide an option for silent yes by default, eg --enable-entra-auth

Reply:
we have -y for silent yes